packages/snowpack/src/index.js

```diff
@@ -2,15 +2,16 @@ import { compareSignatures, isComponent } from '@prefresh/utils';
 
 export default function preactRefreshPlugin(config, pluginOptions) {
 	return {
-		knownEntrypoints: ['@prefresh/core'],
-		async transform({ contents, urlPath, isDev }) {
-			if (!isDev || !urlPath.endsWith('.js') || config.devOptions.hmr === false)
-				return;
+		name: '@prefresh/snowpack',
+		input: ['.js'],
+		output: ['.js'],
```

Comment:
To read more about these changes, please see the in-progress docs: https://snowpack-git-drwpow-plugin-docs.pikapkg.vercel.app/plugins